src/js/profile/mobile/widget/TimePicker.js

```diff
@@ -146,20 +159,31 @@
 				if (name === "hour") {
 					if (self.options.format === "24") {
 						options.min = 0;
-						options.max = 24;
+						options.max = 23;
 					} else {
-						options.min = 0;
+						options.min = 1;
 						options.max = 12;
 					}
 					spinContainer.classList.add(TimePicker.classes.HOUR_CONTAINER);
 				} else if (name === "minute") {
 					options.min = 0;
-					options.max = 60;
+					options.max = 59;
 					spinContainer.classList.add(TimePicker.classes.MINUTE_CONTAINER);
 				}
 
+				spinInput.min = options.min;
+				spinInput.max = options.max;
+				spinInput.type = "number";
+				spinInput.step = "1";
```

Comment:
What's mean the "step"?

Reply:
The step attribute specifies the legal number intervals for a number field.
Example: if step="3", legal numbers could be -3, 0, 3, 6, etc.